.github/workflows/crowdin-sync.yml

```diff
@@ -49,3 +50,31 @@ jobs:
           CROWDIN_PROJECT_ID: ${{ vars.CROWDIN_PROJECT_ID }}
           CROWDIN_API_TOKEN: ${{ secrets.CROWDIN_API_TOKEN }}
           CROWDIN_BASE_URL: ${{ vars.CROWDIN_BASE_URL }}
+  backchannel-comments:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request_review_comment' && contains(github.event.pull_request.head.ref, 'l10n_crowdin') && (contains(github.event.comment.path, '.po') || contains(github.event.comment.path, '.properties')) }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Checkout reusable workflows repo for common scripts
+        uses: actions/checkout@v4
+        with:
+          clean: false
+          repository: warp-ds/reusable-workflows
+          path: temp-reusable-workflows
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - name: npm install
```

Comment:
Could this be changed to pnpm instead?

Reply:
The reason why we didn't go for pnpm here is because this workflow is practically a copy of Finn's crowdin-integration workflow: https://github.schibsted.io/finn/crowdin-integration/blob/main/.github/workflows/crowdin-verify-and-sync.yml
We felt it's easier to test and maintain this workflow when we can map it to the original as much as possible, and not introduce a change to the setup that might potentially break some functionality in the future, e.g. because of some version incompatibilities. I believe we'll eventually have just 1 crowdin-sync reusable workflow for both enterprise and open source apps on Github, which would then be maintained in one place. But until that's settled, I'd rather keep the two as much aligned as possible. Wdyt?